list_of_projects_converter.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+file = open('list_of_projects.txt', 'w')
+
+filecut = os.listdir('./_projects')
+filecut.remove('.placeholder')
```

Comment:
You better have a white list instead of a black list: only consider `.md` files instead of excluding.